ddl/index.go

```diff
@@ -2001,6 +2074,38 @@ func (w *worker) updateReorgInfoForPartitions(t table.PartitionedTable, reorg *r
 	return false, errors.Trace(err)
 }
 
+func runBackfillJobsWithLightning(d *ddl, sess *session, bfJob *BackfillJob, jobCtx *JobContext) error {
+	// TODO: Consider redo it.
+	bc, ok := ingest.LitBackCtxMgr.Load(bfJob.JobID)
+	if ok && bc.Done() {
+		return errors.New(ingest.LitErrGetBackendFail)
+	}
```

Comment:
Is it dead code? You always "unregister" the context first, then setting the status to "done".